libs/designsystem/src/lib/components/modal/modal-wrapper/modal-wrapper.component.scss

```
@@ -34,7 +34,7 @@
   }
 }
 
-:host-context(ion-modal:not(.full-height)) {
+:host-context(ion-modal:not(.kirby-modal--full-height)) {
```

Please align with existing naming:
```suggestion
:host-context(ion-modal:not(.kirby-modal-full-height)) {
```